AsyncDisplayKit/ASDisplayNode.mm

```diff
@@ -2085,11 +2085,7 @@ - (void)setInterfaceState:(ASInterfaceState)newState
   BOOL wasVisible = ASInterfaceStateIncludesVisible(oldState);
 
   if (nowVisible != wasVisible) {
-    if (nowVisible) {
```

Comment:
lol.  This was planning for something, but indeed it never happened ;).  Good change.